include/bout/field.hxx

```diff
@@ -178,6 +178,18 @@ inline bool areFieldsCompatible(const Field& field1, const Field& field2) {
 #define ASSERT1_FIELDS_COMPATIBLE(field1, field2) ;
 #endif
 
+template <typename F>
+inline bool isFci(const F& f) {
```

Comment:
Maybe `canBeFieldAligned`? Not sure.

Also, should this be a member function instead? Then it wouldn't need to be templated. If it should really be a free function, it would be good to add a `static_assert` (see `emptyFrom` etc) to give a nicer error message if passed the wrong type.

Reply:
> Maybe `canBeFieldAligned`? Not sure.

But I want to know whether it is FCI. If we ever have a different, non-field aligned, non FCI method, we might need to reconsider, but I think isFci (or isFCI) is better, as you can use BOUT++ also for non magnetic simulations, and that is also not field aligned, but not FCI either.

> Also, should this be a member function instead? Then it wouldn't need to be templated. If it should really be a free function, it would be good to add a `static_assert` (see `emptyFrom` etc) to give a nicer error message if passed the wrong type.

Good point, will make it a member function :+1: